docs/localdev.md

```diff
@@ -0,0 +1,9 @@
+<!--
+    Copyright 2018 VMware, Inc.
```

Comment:
```suggestion
    Copyright 2019 VMware, Inc.
```

Reply:
Looks like I still need to get used to it!